cmd/helm/init.go

```diff
@@ -117,6 +120,7 @@ func newInitCmd(out io.Writer) *cobra.Command {
 	f.BoolVar(&i.dryRun, "dry-run", false, "do not install local or remote")
 	f.BoolVar(&i.skipRefresh, "skip-refresh", false, "do not refresh (download) the local repository cache")
 	f.BoolVar(&i.wait, "wait", false, "block until Tiller is running and ready to receive requests")
+	f.IntVar(&i.timeout, "timeout", 10, "number of seconds to wait for tiller before timing out")
```

Comment:
Is 10 not low? Have you looked at how long it takes it to startup when you have a lot of releases?

Reply:
This is starting tiller.  I thought you start the tiller and then deploy releases.